### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by maksym-nyrka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,17 @@
-## Node set-up
+# GGX Chain node set-up
 
-> **NOTE:** Here you can find manual on how to run a node locally.
+> **NOTE:** Here you can find manual on how to run a node locally.
```
Comment (obsessed-cake) [inline README.md:3]:
```suggestion
> **NOTE:** Here you can find manual on how to run a node locally.
```

Comment (maksym-nyrka) [reply README.md:3]: That's for a new line

Comment (obsessed-cake) [reply README.md:3]: ok

**Review by uchu — APPROVED** [review]
LGTM

**Pull request merged**